Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN apk add --no-cache \
     libsass-dev \
     make \
     musl-dev \
+    jq \
```

Comment:
Why do we need `jq`? Is it currently missing and therefore failing because of it? Or is it for future work?

Reply:
I use it here [build-image-manifest.sh](https://github.com/weaveworks/eksctl/blob/master/build-image-manifest.sh#L13) and we use it in circleci to check if the manifest is up to date.

Reply:
Thanks for the confirmation! ✨